scripts/deployment/maticOracle.json

```diff
@@ -0,0 +1,67 @@
+{
+    "oracles" : [
+        {
+            "targetPairName":"DAI/MATIC",
+            "tokenPairName":"DAI/ETH",
+            "tokenPriceOracle":"0xFC539A559e170f848323e19dfD66007520510085",
+            "maticPairName":"MATIC/ETH",
+            "maticPriceOracle":"0x327e23A4855b6F663a28c5161541d69Af8973302"
+        },
+        {
+            "targetPairName":"USDC/MATIC",
+            "tokenPairName":"USDC/ETH",
+            "tokenPriceOracle":"0xefb7e6be8356cCc6827799B6A7348eE674A80EaE",
+            "maticPairName":"MATIC/ETH",
+            "maticPriceOracle":"0x327e23A4855b6F663a28c5161541d69Af8973302"
+        },
+        {
+            "targetPairName":"USDT/MATIC",
+            "tokenPairName":"USDT/ETH",
+            "tokenPriceOracle":"0xf9d5AAC6E5572AEFa6bd64108ff86a222F69B64d",
+            "maticPairName":"MATIC/ETH",
+            "maticPriceOracle":"0x327e23A4855b6F663a28c5161541d69Af8973302"
+        },
+        {
+            "targetPairName":"WBTC/MATIC",
+            "tokenPairName":"WBTC/ETH",
+            "tokenPriceOracle":"0xA338e0492B2F944E9F8C0653D3AD1484f2657a37",
+            "maticPairName":"MATIC/ETH",
+            "maticPriceOracle":"0x327e23A4855b6F663a28c5161541d69Af8973302"
+        },
+        {
+            "targetPairName":"SUSHI/MATIC",
+            "tokenPairName":"SUSHI/USD",
+            "tokenPriceOracle":"0x49B0c695039243BBfEb8EcD054EB70061fd54aa0",
+            "maticPairName":"MATIC/USD",
+            "maticPriceOracle":"0xAB594600376Ec9fD91F8e885dADF0CE036862dE0"
```

Comment:
Same for SUSHI, LINK, CRV, QUICK, ETH

Reply:
Chainlink has only USD price aggregator for these tokens. So we take MATIC/USD price feed to convert to token price in terms of MATIC

Reply:
Cool